src/features/integration/components/form/ApplicantForm.tsx

```diff
@@ -0,0 +1,58 @@
+import {FormControl, FormGroup, InputLabel, MenuItem, Select, SelectChangeEvent, TextField} from '@mui/material';
+import React from 'react';
+import {dropdownPlaceholder} from "../../util/DefaultValues";
+
+const ApplicantForm: React.FunctionComponent<any> = (props) => {
+    return (
+        <FormGroup className={props.style.formControl}>
+            <FormControl>
```

Comment:
Similar setup on many components. Create a component for this?

Reply:
Inputfield component used in all form components